doc/classes/SoftBody3D.xml

```diff
@@ -99,10 +105,10 @@
 		<member name="ray_pickable" type="bool" setter="set_ray_pickable" getter="is_ray_pickable" default="true">
 			If [code]true[/code], the [SoftBody3D] will respond to [RayCast3D]s.
 		</member>
-		<member name="simulation_precision" type="int" setter="set_simulation_precision" getter="get_simulation_precision" default="0">
+		<member name="simulation_precision" type="int" setter="set_simulation_precision" getter="get_simulation_precision" default="5">
 			Increasing this value will improve the resulting simulation, but can affect performance. Use with care.
 		</member>
-		<member name="total_mass" type="float" setter="set_total_mass" getter="get_total_mass" default="0.0">
+		<member name="total_mass" type="float" setter="set_total_mass" getter="get_total_mass" default="1.0">
```

Comment:
Don't include unrelated changes caused by doctool in the commit. (Use `git add -p` and `git reset -p` to make partial commits.)

The same applies to other parts modified in this file.